Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -54,6 +54,22 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
         return "https://www.mediawiki.org/wiki/Special:MyLanguage/Wikimedia_Apps/iOS_FAQ?uselang=\(languageCode ?? "en")#Editing"
     }
     
+    private func noncontributorTitle() -> String {
+        // Check if current date is before the temporary date (January 1, 2026)
+        let calendar = Calendar.current
+        var expiryDateComponents = DateComponents()
```

Comment:
Minor - might be good to have this expiry date components in a shared computed variable so it's only defined once.